interceptor/go.mod

```diff
@@ -0,0 +1,10 @@
+module interceptor
+
+go 1.15
+
+require (
+	github.com/pion/rtcp v1.2.4
+	github.com/pion/rtp v1.6.1
+	github.com/pion/webrtc-v3-design/webrtc v0.0.0-20201018190645-01b2dee52ae4
```

Comment:
I would put these in `webrtc/pkg/interceptor` for now.

It would be interesting to move them out of tree eventually. I just don't see a good way to do it yet.

When we do it will be easy enough to alias though!